ci/terraform/oidc/build.tfvars

```diff
@@ -29,4 +29,5 @@ AZUx4RCDu+VWAZpPi1NaF5XWvkFNFwH+MyLkATh90UEJDe+ayKW6AXFcRQ==
 -----END PUBLIC KEY-----
 EOT
 
-orch_client_id = "orchestrationAuth"
\ No newline at end of file
+orch_client_id    = "orchestrationAuth"
```

Comment:
There's a few changes which are just formatting. Separate into another commit?

Reply:
This is a result of the line below being added and tf fmt being run. It aligns the equals signs, so these are related changed and need to  be in the same commit